packages/core/contracts-ovm/insured-bridge/external/iAVM_Inbox.sol

```diff
@@ -0,0 +1,36 @@
+// Copied mostly from https://github.com/makerdao/arbitrum-dai-bridge/blob/7f1b47ef65a43f1696c5f1681109daac127d9c95/contracts/arbitrum/IInbox.sol
+// - Bumped solidity version to >= 0.8.x
+// - Removed functions from interface that are not used by AVM_CrossDomainEnabled.sol
+// - Removed IMessageProvider inheritance.
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+ * Copyright 2021, Offchain Labs, Inc.
```

Comment:
do we need to include this license? if you reference where it came from do we need this bit?

Reply:
Removed